components/ActionSheet/examples/ActionSheetTypicalUseExampleViewController.m

```diff
@@ -44,15 +44,7 @@ - (instancetype)init {
 - (void)viewDidLoad {
   [super viewDidLoad];
 
-  id<MDCColorScheming> colorScheme;
-  if (self.containerScheme.colorScheme != nil) {
-    colorScheme = self.containerScheme.colorScheme;
-  } else {
-    colorScheme =
-        [[MDCSemanticColorScheme alloc] initWithDefaults:MDCColorSchemeDefaultsMaterial201804];
-  }
-
-  self.view.backgroundColor = colorScheme.backgroundColor;
+  self.view.backgroundColor = self.containerScheme.colorScheme.backgroundColor;
```

Comment:
I think checking if self.containerScheme is nil is still required (and assigning it a default container-scheme if it's nil)), because it may not be injected.
This is true for all examples.

Reply:
This has been updated.